tests/integration/commands/test_get.py

```diff
@@ -1,11 +1,55 @@
+import random
+from string import ascii_letters
+
+import confluent_kafka
 import pytest
 from click.testing import CliRunner
 
-from esque.cli.commands import get_topics
+from esque.cli.commands import get_topics, get_consumergroups, get_brokers
+from esque.controller.topic_controller import TopicController
+from esque.resources.topic import Topic
+
+
+@pytest.mark.integration
+def test_smoke_test_get_topics(non_interactive_cli_runner: CliRunner):
+    result = non_interactive_cli_runner.invoke(get_topics, ["--verbose"])
+    assert "No config provided in" not in result.output
+    assert result.exit_code == 0
+
+
+@pytest.mark.integration
+def test_get_topics_with_prefix(
+    non_interactive_cli_runner: CliRunner,
+    topic_controller: TopicController,
+    confluent_admin_client: confluent_kafka.admin.AdminClient,
+):
+    topic_base = "".join(random.choices(ascii_letters, k=5))
+    prefix_1 = "ab"
+    prefix_2 = "fx"
+    new_topics = [prefix_1 + topic_base, prefix_2 + topic_base, prefix_1 + prefix_2 + topic_base]
+    for new_topic in new_topics:
+        topic_controller.create_topics([Topic(new_topic, replication_factor=1)])
+    confluent_admin_client.poll(timeout=1)
+
+    result = non_interactive_cli_runner.invoke(get_topics, ["-p", prefix_1])
+
+    assert result.exit_code == 0
+    retrieved_topics = result.output.split("\n")
```

Comment:
You can use your own new feature here. Output it as json and parse it to get the exact list of topics.